src/store/actions.js

```diff
@@ -113,8 +113,9 @@ export default {
         commit("setNotification", "Error getting request data from server")
       })
   },
-  retrievePostsByNegotiationId({ state, commit }, { negotiationId }) {
-    return axios.get(`${NEGOTIATION_PATH}/${negotiationId}/posts`, getBearerHeaders(state.oidc.access_token))
+  retrievePostsByNegotiationId({ state, commit }, { negotiationId, type, resourceId }) {
+    let url = resourceId ? `${NEGOTIATION_PATH}/${negotiationId}/posts?type=${type}&resource=${resourceId}` : `${NEGOTIATION_PATH}/${negotiationId}/posts?type=${type}`
```

Comment:
it's cleaner (and probably safer) to use `params` argument of axios.get passing an object with query params instead of adding them to the url. 

```
let params = {
  type: type
}
...
```

See https://axios-http.com/docs/req_config

Reply:
Fixed in 338765dbdacd1bacd4779ccc2b2d75595643e489 and in 1b21a66bbac3852903aaa45e474ea307313f9480